recipes-kernel/linux/linux-yocto-onl_5.15.bb

```diff
@@ -39,7 +39,7 @@ DEPENDS += "gmp-native libmpc-native"
 # With i2c-i801 and i2c-ismt built as modules there is no guarantee one or the
 # other will be probed first and gets bus 0, but platforms use hardcoded paths
 # assuming bus 0 is i2c-i801.
-# So mark i2c-i801 as to be loaded before i2c-ismt to make sure it is probed
-# first.
+# So blacklist i2c-i801 to load it via platform code once i2c-ismt finished
+# registering its i2c bus.
 KERNEL_MODULE_PROBECONF:append:x86-64 = " i2c-ismt"
-module_conf_i2c-ismt = "softdep i2c-ismt pre: i2c-i801"
+module_conf_i2c-ismt = "blacklist i2c-ismt"
```

Comment:
The code looks correct, but the comment describes something different. It is i2c-ismt which is blacklisted, and it needs to wait for i2c-801 to finish registering its i2c bus.

Reply:
Indeed, did a thinko there. Fixed.